projects/ngneat/forms-manager/src/lib/forms-manager.ts

```diff
@@ -389,6 +391,25 @@ export class NgFormsManager<FormsState = any> {
     return this;
   }
 
+  private getValueChangeStreams(control: AbstractControl) {
```

Comment:
Can you explain the solution, please?

Reply:
This function returns an array of valueChanges streams. If a control has updateOn=blur we don't want to do the debounce so we map it to NO_DEBOUNCE.